.eslintrc

```diff
@@ -0,0 +1,38 @@
+{
+  "env": {
+    "node": true
+  },
+  "globals": {},
+  "rules": {
+    "no-bitwise": 2,
+    "camelcase": 0,
+    "curly": 2,
+    "eqeqeq": 2,
+    "no-unused-expressions": [
+      2,
+      "allowTernary": true
```

Comment:
Ignore me, it is still supposed to be a valid JSON object, so this does need to be wrapped